eventio/iact/parse_corsika_data.py

```diff
@@ -155,99 +174,109 @@ def parse_corsika_run_header(run_header):
     'observation_level_curvature_flag',
 ])
 
+
 def parse_corsika_event_header(event_header):
     h = event_header
-    
+
     n_random_number_sequences = int(round(h[12]))
     if (n_random_number_sequences < 1) or (n_random_number_sequences > 10):
-        raise ValueError('Number of random number sequences n must be 0 < n < 11, but is: '+str(h[12]))
-    
+        raise ValueError(
+            'Number of random seed sequences must be 0 < n < 11, but is: {}'.format(h[12])
+        )
+    seed_info = h[13:13 + 3 * n_random_number_sequences].reshape(
+        n_random_number_sequences, 3
+    )
+
     n_obs_levels = int(round(h[46]))
     if (n_obs_levels < 1) or (n_obs_levels > 10):
-        raise ValueError('number of observation levels n must be 0 < n < 11, but is: '+str(h[46]))
-    
-    n_reuse=int(h[97])
+        raise ValueError(
+            'number of observation levels n must be 0 < n < 11, but is: {}'.format(h[4])
+        )
+
+    n_reuse = int(h[97])
     core_x = h[98:98+n_reuse],
     core_y = h[118:118+n_reuse],
 
-    return CorsikaEventHeader(    
-        event_id = int(round(h[1])),
```

Comment:
Combining lots of whitespace improvements together with actual new features in a single commit is suboptimal from the reviewers point of view.